tracing-mock/src/subscriber.rs

```diff
@@ -1,9 +1,124 @@
-#![allow(missing_docs, dead_code)]
+//! An implementation of the [`Subscribe`] trait to receive and validate
+//! `tracing` data.
+//!
+//!
+//! The [`MockSubscriber`] is the central component in these tools. The
```

Comment:
IMO it's not _super_ obvious what "these tools" is referring to. maybe 
```suggestion
//! The [`MockSubscriber`] is the central component of this crate. The
```
or
```suggestion
//! The [`MockSubscriber`] is the central component of `tracing-mock`. The
```